source/faq.txt

```diff
@@ -20,20 +20,18 @@ Support.
 Can I perform reads or writes to my destination cluster while ``mongosync`` is syncing?
 ---------------------------------------------------------------------------------------
 
-You can perform reads during synchronization if ``canCommit`` is ``true``. 
-However, the data that you read is 
-:term:`eventually consistent <eventual consistency>`, meaning that you might not 
-always read the latest writes.
+You can perform reads during synchronization at any time. However, the data 
+that you read is :term:`eventually consistent <eventual consistency>`, meaning 
```

Comment:
I think the "meaning that you might not always read the latest writes" can be removed altogether, since the consistency link provides the full context on what reads might look like pre-commit.

Reply:
That makes sense! Updating.

Reply:
EDIT: I'm leaning towards leaving it as is since users might be skimming through this FAQ quickly and might not want to always click into the glossary entry for `eventually consistent` and be routed to a different set of docs pages (the server manual, which is where the glossary lives).

Reply:
thanks! I'd push for removing that text since it's not technically accurate - we've seen folks who have interpreted this to mean that stopping the source cluster mid-sync just means that the destination has a stale snapshot of the source, which isn't the case.

Reply:
gotcha! i'll go ahead and remove then.